docs/Toolchains.md

```diff
@@ -6,28 +6,6 @@ When you call `nodejs_register_toolchains()` in your `WORKSPACE` file it will se
 
 If you have an advanced use-case and want to use a version of node not supported by this repository, you can also register your own toolchains.
 
-## Node.js binary for the target platform
```

Comment:
This documentation dates back to when rules_nodejs had more than just toolchains